client/common/client.c

```diff
@@ -68,6 +69,19 @@
 #include <freerdp/utils/aad.h>
 #endif
 
+#ifdef WITH_SSO_MIB
+#include "sso_mib_tokens.h"
+
+enum sso_mib_state
+{
+	SSO_MIB_STATE_INIT = 0,
+	SSO_MIB_STATE_FAILED = 1,
+	SSO_MIB_STATE_SUCCESS = 2,
+};
+
+static enum sso_mib_state sso_mib_state = SSO_MIB_STATE_INIT;
```

Comment:
should this not be part of `instance/context` too?

Reply:
Yes, I can move it over (possibly into the Wrapper struct)